.gitignore

```diff
@@ -4,4 +4,5 @@ vendor
 yarn.lock
 dist
 .vscode
-.gobincache
\ No newline at end of file
+.gobincache
+.idea/*
```

Comment:
please put those into your global gitignore file, not in the repo. I know we have `.vscode` in there, but this should not prove as an example.

Reply:
✔️